Doc/library/zipfile.rst

```diff
@@ -183,6 +184,9 @@ ZipFile Objects
       Previously, a plain :exc:`RuntimeError` was raised for unrecognized
       compression values.
 
+   .. versionchanged:: 3.6.2
```

Comment:
The policy is described here:  https://mail.python.org/pipermail/python-committers/2017-March/004290.html